.gitmodules

```diff
@@ -383,3 +383,12 @@
 [submodule "frozen/Adafruit_CircuitPython_SHT4x"]
 	path = frozen/Adafruit_CircuitPython_SHT4x
 	url = https://github.com/adafruit/Adafruit_CircuitPython_SHT4x
+[submodule "frozen/Adafruit_CircuitPython_NTP"]
+	path = frozen/Adafruit_CircuitPython_NTP
+	url = https://github.com/adafruit/Adafruit_CircuitPython_NTP.git
+[submodule "frozen/Adafruit_CircuitPython_PCF8563"]
+	path = frozen/Adafruit_CircuitPython_PCF8563
+	url = https://github.com/adafruit/Adafruit_CircuitPython_PCF8563.git
+[submodule "frozen/CircuitPython_BMA423"]
+	path = frozen/CircuitPython_BMA423
+	url = https://github.com/jposada202020/CircuitPython_BMA423.git
```

Comment:
Please don't add more frozen libraries unless you need them to free up RAM. Libraries outside of the image are much easier to update.

Reply:
Saving RAM is the intent of freezing these libraries, I've only frozen fairly stable libraries required by the hardware; but if you require me to take out the frozen libraries I will.

Reply:
Are you saving ram preemptively or did you have demo code already need this? I'd rather only add it when it is needed.

Reply:
Mostly from practical testing. I have had this firmware for some time and just decided to update it to CP 9 and commit.
Originally I didn't freeze anything, but by the time I added all the hardware libraries, ePaper display driver, graphics libraries and a bitmap for the watch face it was starting to run low on memory. The perils of ESP32 without PSRAM I suppose. I'll let you be the judge on whether this is sufficient justification to add frozen libraries.

Reply:
Ya, I think that's enough reason.